include/capstone/loongarch.h

```diff
@@ -0,0 +1,2516 @@
+#ifndef CAPSTONE_LOONGARCH_H
+#define CAPSTONE_LOONGARCH_H
+
+/* Capstone Disassembly Engine */
+/* By Jiajie Chen <c@jia.je> 2024 */
+/*    Yanglin Xun <1109673069@qq.com>, 2024 */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if !defined(_MSC_VER) || !defined(_KERNEL_MODE)
+#include <stdint.h>
+#endif
+
+#include "platform.h"
+#include "cs_operand.h"
+
+/// Operand type for instruction's operands
+typedef enum loongarch_op_type {
+	LoongArch_OP_INVALID = CS_OP_INVALID, ///< Invalid
```

Comment:
all other architectures have the same naming convention, that is to use capitalized letters for enums. 
can you change this to follow that?